src/DotNetWorker.Grpc/GrpcCollectionModelBindingData.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc.Messages
+{
+    internal partial class GrpcCollectionModelBindingData : Microsoft.Azure.Functions.Worker.Core.CollectionModelBindingData
```

Comment:
I think initially we discussed to follow the existing pattern similar to `IHttpCookie` - [Link](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/src/DotNetWorker.Core/Http/IHttpCookie.cs). Hence, we used `partial class` similar to [this](https://github.com/Azure/azure-functions-dotnet-worker/blob/00d4386c9ce5ed7c61f9ea75ef6ddaeaea2cd157/src/DotNetWorker.Grpc/Http/RpcHttpCookie.cs#L9). Then, based on the discussion in [previous similar PR](https://github.com/Azure/azure-functions-dotnet-worker/pull/1107#discussion_r1002146117), we updated our pattern to follow `HttpRequestData` ([Link](https://github.com/Azure/azure-functions-dotnet-worker/blob/00d4386c9ce5ed7c61f9ea75ef6ddaeaea2cd157/src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs#L15)) i.e. to use abstract class instead of an interface, so I feel we should not need partial class now.

E2E seems working even after removing partial keyword. @liliankasem let me know your thoughts on this change. It has been done both on `GRPCModelBindingData` and `GRPCCollectionModelBindingData`.

Reply:
Yeah I thought we had a very specific reason for it to be partial that may have been related to GRPC. I'd like to get Fabio to weigh in as it was his suggestion originally to make this partial so I'm leaning more to keep it for now